Source/WebKit/UIProcess/mac/WebPageProxyMac.mm

```diff
@@ -364,6 +364,10 @@ static inline bool expectsLegacyImplicitRubberBandControl()
 
 bool WebPageProxy::acceptsFirstMouse(int eventNumber, const WebKit::WebMouseEvent& event)
 {
+    const int synthesizedMouseEventMagicEventNumber = 0;
```

Comment:
If we are going to start using this outside of WebAutomationSessionMac we probably want to to instead move the declaration of the original static constant to a header like `WebAutomationSession.h` so we only declare it once, and give is a new name to make it clear it is used for automation, since that isn't clear here. Maybe `webAutomationSynthesizedMouseEventMagicEventNumber`